android/app/build.gradle

```diff
@@ -125,7 +125,7 @@ dependencies {
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
 
     //Branch integration for deep linking
-    compile ('io.branch.sdk.android:library:2.5.9') {
+    compile('io.branch.sdk.android:library:2.5.9') {
```

Comment:
Please remove these unnecessary changes.

You can look through the PR and see which changes are necessary and which ones are not

Like, there is no need for a build.gradle to change in this PR